src/Compliance/Consent/ConsentManager.php

```
@@ -0,0 +1,427 @@
+            ->get();
+    }
+
+    /**
+     * Send reconsent notifications.
+     */
+    public function notifyReconsent( string $purpose ): int
+    {
+        $records = $this->getUsersRequiringReconsent( $purpose );
+        $count   = 0;
+
+        foreach ( $records as $record ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$record'. (undefined)

(UnusedLocalVariable)